.github/workflows/luis_ci.yaml

```diff
@@ -86,14 +86,23 @@ jobs:
     - name: Ludown to LUIS model
       run: bf luis:convert -i $LU_FILE -o ./model.json --name 'LUIS CI pipeline - ${{ github.run_id }}' --versionid $LUISAppVersion
 
-    # When doing a merge to master, use the master LUIS app - create if necessary (soft fails if exists)
+    # When doing a merge to master, use the master LUIS app
     - name: Get master LUIS application ID
       run: |
-        bf luis:application:create --name $LUIS_MASTER_APP_NAME --subscriptionKey $LUISAuthoringKey  --endpoint $LUISAuthoringEndpoint --versionId=0.1
```

Comment:
How does this change guarantee that the app already exists at this stage?

Reply:
@AmirGeorge  creating the application is moved to the next step and is only executed if the application is not found in this step which tries to find the application by its name